contracts/Whitelist.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { IWhitelist } from "./interfaces/IWhitelist.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Whitelist is IWhitelist, Ownable {
+
+    mapping(address => uint) whitelist;
+
+    function isWhitelisted(address user) public view returns(bool) {
+        return whitelist[user] > 0;
+    }
+    
+    function allowedAllocation(address user) public view returns(uint) {
+        return whitelist[user];
+    }
+
+    function updateAllocation(address user, uint allocation) public onlyOwner {
+        whitelist[user] = allocation;
+    }
+}
```

Comment:
We need to get rid of this error, you can probably set appropriate setting in your IDE, example for VSCode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files#:~:text=Additionally%2C%20in%20the%20Settings%20page,there%20isn't%20already%20one.